src/h5web/vis-packs/core/utils.ts

```diff
@@ -42,11 +41,20 @@ const adaptedLogTicksThreshold = scaleLinear({
 });
 
 export function createAxisScale(
-  config: PickScaleConfig<ScaleType, number>
+  scaleParams: ScaleParams,
+  config: VisxScaleConfig | GammaScaleConfig
 ): AxisScale {
-  const { type } = config;
+  if (isScaleType(scaleParams) && scaleParams !== ScaleType.Gamma) {
```

Comment:
If you switch to `VisScaleType`, then don't forget to rename `scaleParams` to `scaleType` here. Also, I think `Gamma` is not possible here, since we omit it from the type.